player/lua/stats.lua

```diff
@@ -755,6 +755,6 @@ mp.register_event("video-reconfig",
         end
     end)
 
-if mp.get_property("osd-stats") then
+if mp.get_property_bool("osd-stats") then
```

Comment:
Probably should not have made this an extra commit. Just edit the previous commit then force push.